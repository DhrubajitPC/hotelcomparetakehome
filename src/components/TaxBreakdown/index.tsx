import React, { FC } from "react";

type Props = {
  tax: number;
  hotelFees: number;
};

const TaxBreakdown: FC<Props> = ({ tax, hotelFees }) => (
  <>
    <p>Tax: {tax}</p>
    <p>Hotel Fees: {hotelFees}</p>
  </>
);

export default TaxBreakdown;
