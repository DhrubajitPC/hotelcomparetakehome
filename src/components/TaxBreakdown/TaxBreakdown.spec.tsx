import { render } from "@testing-library/react";
import TaxBreakdown from ".";
import React from "react";

describe("#TaxBreakdown", () => {
  it("should render correctly", () => {
    const { container } = render(<TaxBreakdown tax={20} hotelFees={100} />);
    expect(container).toMatchSnapshot();
  });
});
