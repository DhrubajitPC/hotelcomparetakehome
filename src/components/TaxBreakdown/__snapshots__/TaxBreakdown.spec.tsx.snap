// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaxBreakdown should render correctly 1`] = `
<div>
  <p>
    Tax: 
    20
  </p>
  <p>
    Hotel Fees: 
    100
  </p>
</div>
`;
